index.html

```diff
@@ -271,7 +271,7 @@ <h2 class="additional-info__title">We know a lot</h2>
         </ul>
       </section>
 
-      <section class="donations">
+      <section class="donations" id="donations">
```

Comment:
I think that in BEM we don't use id's.

Reply:
We don't use IDs for styling, but I don't know of another way to make a local link.